two_factor/templates/two_factor/_wizard_forms.html

```diff
@@ -1,10 +1,22 @@
 {{ wizard.management_form }}
 {{ wizard.form.non_field_errors }}
 
-{% for field in wizard.form %}
-    <div class="form-field {% if field.errors %}form-field--error{% endif %}">
-        {{ field.errors }}
-        <label>{{ field.label }}</label>
-        {{ field }}
-    </div>
-{% endfor %}
+{% if wizard.steps.current == 'auth' %}
+      <div class="form-row">
+        {{ form.username.errors }}
+        {{ form.username.label_tag }} <input type="text" name="auth-username" autofocus="" required="" id="id_username">
+      </div>
+      <div class="form-row">
+        {{ form.password.errors }}
+        {{ form.password.label_tag }} <input type="password" name="auth-password" autocomplete="current-password" required="" id="id_password">
+        <input type="hidden" name="next" value="{{ next }}">
+      </div>
+{% else %}
```

Comment:
This template is a generic template used by all wizard forms not only by login form. I don't think that putting specific values and fields of login from here is a good idea. 
Actually I think that the default fields styling is ok and this can be skipped completely. 
1. 
But if you want to change the fields markup for the particular step of the particular form you can do one of the following options:
* Add template for this particular form which inherits from this template
* Add templates for every step of this form